lexicons/app/bsky/purchase/getFeatures.json

```diff
@@ -0,0 +1,33 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.purchase.getFeatures",
+  "defs": {
+    "main": {
+      "type": "query",
+      "description": "Enumerate unlocked paid feature flags for the requesting account. Requires auth.",
+      "output": {
```

Comment:
let's put empty `input`s on all these schemas

it's a little goofy but basically going from "no input" to "some input" is in this lexicon gray area. so this is future proofing it to the possible addition of some input in the future